lib/cb/models/implementations/education.rb

```diff
@@ -9,8 +19,8 @@ def initialize(args = {})
         @code          = args['Code'] || ''
         @language			 = args['Name']['@language'] unless args['Name'].nil?
         @text          = args['Name']['#text'] unless args['Name'].nil?
-        @language      ||= ''
-        @text          ||= ''
+        @language ||= ''
+        @text ||= ''
```

Comment:
Oh, I guess it does this spacing removal when there's a `||` in front of the equal sign, but not when it's a plain equal. Interesting.